charts/ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml

```diff
@@ -33,24 +33,24 @@ spec:
       containers:
         - name: driver-registrar
           image: "{{ .Values.nodeplugin.registrar.image.repository }}:{{ .Values.nodeplugin.registrar.image.tag }}"
+          imagePullPolicy: {{ .Values.nodeplugin.registrar.image.pullPolicy }}
           args:
             - "--v=5"
             - "--csi-address=/csi/{{ .Values.socketFile }}"
             - "--kubelet-registration-path={{ .Values.socketDir }}/{{ .Values.socketFile }}"
           lifecycle:
             preStop:
               exec:
-                  command: [
-                    "/bin/sh", "-c",
-                    'rm -rf /registration/{{ .Values.driverName }}
+                command: [
+                  "/bin/sh", "-c",
+                  'rm -rf /registration/{{ .Values.driverName | replace " " "" }}
```

Comment:
@wilmardo can you explain how this works?

Reply:
Helm has sprig available for templating functions:
https://github.com/helm/helm/blob/master/docs/chart_template_guide/functions_and_pipelines.md

More about sprig string functions:
http://masterminds.github.io/sprig/strings.html

The replace function replaces a space `" "` with no space `""`

Will add a NOTE above it to explain :)